source/rle.js

```diff
@@ -0,0 +1,19 @@
+const rle = (input) => {
+    if (!(typeof input === 'string' || input instanceof String)) {
+	throw new Error('Parameter is not a string or missing!');
+    }
+    let result = '';
+    let count = 1;
+    for (let i = 1; i <= input.length; i++) {
+	if (input[i] === input[i-1]) {
```

Comment:
Минорная история по форматированию (и ниже тоже поправь, пожалуйста): 

```suggestion
	if (input[i] === input[i - 1]) {
```

Так более читаемо